pkg/schema/openapi.go

```diff
@@ -74,16 +86,26 @@ func (o *OpenAPIDocument) calculateProperties(schemaVal interface{}) *yamlmeta.M
 		if typedValue.String() == "float" {
 			property.Items = append(property.Items, &yamlmeta.MapItem{Key: "format", Value: "float"})
 		}
+		if typedValue.GetDescription() != "" {
+			property.Items = append(property.Items, &yamlmeta.MapItem{Key: "description", Value: typedValue.GetDescription()})
+		}
 		return &property
 	case *NullType:
 		properties := o.calculateProperties(typedValue.GetValueType())
 		properties.Items = append(properties.Items, &yamlmeta.MapItem{Key: "nullable", Value: true})
+		if typedValue.GetDescription() != "" {
+			properties.Items = append(properties.Items, &yamlmeta.MapItem{Key: "description", Value: typedValue.GetDescription()})
+		}
 		return properties
 	case *AnyType:
-		return &yamlmeta.Map{Items: []*yamlmeta.MapItem{
+		m := &yamlmeta.Map{Items: []*yamlmeta.MapItem{
 			{Key: "nullable", Value: true},
 			{Key: "default", Value: typedValue.GetDefaultValue()},
 		}}
+		if typedValue.GetDescription() != "" {
+			m.Items = append(m.Items, &yamlmeta.MapItem{Key: "description", Value: typedValue.GetDescription()})
+		}
+		return m
```

Comment:
We're using two different names for the same variable: `properties` vs. `m`. At this point, it's a difference without a reason (at least that I can see, right now).

Let's make them all consistent.